docs/source/en/_toctree.yml

```diff
@@ -145,6 +143,24 @@
     - local: api/loaders
       title: Loaders
     title: Main Classes
+  - sections:
+    - local: api/models/overview
+      title: Overview
+    - local: api/models/unet
+      title: UNet
+    - local: api/models/vq
+      title: VQModel
+    - local: api/models/autoencoderkl
+      title: AutoencoderKL
+    - local: api/models/transformer2d
+      title: Transformer2D
+    - local: api/models/transformer_temporal
+      title: Transformer Temporal
+    - local: api/models/prior_transformer
+      title: Prior Transformer
+    - local: api/models/controlnet
+      title: ControlNet
```

Comment:
Neat separation!

For UNet, I would further segregate, though:

* `UNet1DModel`
* `UNet2DModel` 
* `Unet2DConditionModel`
* `UNet3DConditionModel`

Do you think it could make more sense to have the segregation like this?